src/main/java/com/flowingcode/vaadin/addons/demo/TabbedDemo.java

```diff
@@ -206,24 +206,37 @@ public void showRouterLayoutContent(HasElement content) {
   public void removeRouterLayoutContent(HasElement oldContent) {
     getElement().removeChild(1);
   }
+  
+  private void updateSplitterPosition() {
+    setSourceVisible(codeCB.getValue());
+  }
 
   public void setSourceVisible(boolean visible) {
-    sourceVisible = visible;
```

Comment:
If `sourceVisible` is removed, please make `setSourceVisible` update `codeCB` (so that its value can be modified programmatically; the demo site relies on that behavior)